text/logs/0114-log-correlation.md

```diff
@@ -0,0 +1,137 @@
+a span identifier. A trace may contain multiple spans, arranged as a tree, and 
+may also contain links to other related spans. The combination of a trace 
+identifier and a span identifier corresponds to a specific scope of work. In the 
+tracing API, that scope can also contain attributes and events that describe 
+what work the program being traced was currently performing.
+
+In most cases when traces and logs are used in tandem, the attributes of the 
+current scope do not need to be added to the log entry- doing so would duplicate 
+transmission of those values. As one span is likely associated with several (or 
+many) log entries, it is more efficient to transmit those attributes with the 
+span once rather than many times with each log entry. As a result, for most 
+purposes the goal is to set the three values of traceid, spanid, and traceflags 
+into each log entry that should be associated with that span.
+
+| Field      | Required | Format  
+| :--------- | :------- | :--------------------------------------------------
+| traceid    | Yes      | 16-byte numeric value as Base16-encoded string
+| spanid     | Yes      | 8-byte value represented as Base16-encoded string
+| traceflags | No       | 8-bit numeric value as a Base16-encoded string
+
+### Resource Correlation
+As important as what was occurring in a program’s execution is, where it was 
+occurring is just as important. Resource correlation allows a log entry to be 
+associated with an infrastructure resource, and in turn system and program 
+metrics that describe the wider program state. The form that the resource takes 
+is also more diverse than the tracing scope- the resource may be a pod running 
+in a Kubernetes cluster, a virtual machine running in a cloud, a serverless 
+lambda, or an old-school server sitting in a data center. An application 
+environment may be an orchestration of multiple types of resource working 
+together.
+
+From a logging standpoint, a resource is also almost always a constant within an 
+application process- a container may not have the same identifier on every run, 
+but it does keep that identifier while it’s running, and that resource 
+identifier is constant for every log entry created on that resource. As a 
+result, resource correlation may not happen at the log entry level, so we may or 
+may not put the resource correlation information in the log entry itself.
+
+Resource information may be managed as part of the log ingestion process- for 
+example a Docker logging driver will know which container logs came from. 
+Full resource information may also not be available, as when logs are 
+aggregated by syslog or a similar system that has less context available to it. 
+As a result resource correlation information in the logs entries themselves 
+should be considered optional. If included, the resource information should 
+follow the semantic conventions for resources.
```

Comment:
It would be good to link to the semantic convention document so that it is clear this refers to OpenTelemetry conventions.